packages/db/src/models/execution.ts

```diff
@@ -13,6 +13,13 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
     });
   }
 
+  create(
+    knexOrTransaction: Knex | Knex.Transaction,
+    item: PostgresExecution
+  ) {
+    return super.create(knexOrTransaction, item, '*');
```

Comment:
Thanks for making this change across the other types. I was initially concerned because `collectionPgModel.create()` was updated with this change before the others but as long as we're consistent I think it's great. It might be worth making the same change to `providerPgModel.create()` and `asynOperationPgModel.create()` in this PR? It doesn't look like it's explicitly part of the scope here, it would just be for consistency. Thoughts?

Also, just curious. The return of this method is an array. e.g.

```
const [executionPgRecord] = await executionPgModel.create(
    knex,
    execution
  );
```
Could we instead find a way to just return a single record so that we don't have to index into an array every time? I think our `create` logic would only ever create one record as we have it. Eh?

Reply:
I would agree that those changes would be out of scope here. I'm also not 100% sure if we even need them for providers and async operations.

We needed them here because we wanted to be able to return the entire record on PG write, so that we can use it as the basis for the SNS message and we don't have to immediately look it up after write (causing another database query). 

But we don't send providers/async operations to any SNS topics, so I don't see any obvious need for those changes. 

The intent of the refactor was also to preserve behavior of existing models precisely so we could avoid other breaking changes, so absent a compelling reason to change them I'd be inclined to leave them as is.

Reply:
Yup, totally on board with limiting the scope on this PR. Especially because the change I'm suggesting is really just for consistency. I _do_ like avoiding the array return type though if we're going to override the base model's `create` method anyway. It feels like that could change without too much fuss.

It does also seem like it would make sense to have all of the `create` methods return the same type (in a separate PR) but you're right, there aren't instances I can find where we're looking up a record immediately after `create` _except for the test helpers_.

 On the one hand this is why we extend a `Base` model, so that we can have e.g. `Collection` implement a different `create` method. On the other hand I think it would be really easy to make this same change to all of the models (or really, just the base model's `create` at that point). It looks like it wouldn't even break any logic. It would just require test updates.

Reply:
Yeah I'm not against the change in practice, but it would require refactoring a bunch of tests at least and possibly other code I'm not thinking of, so just lots of lines of code. But happy to see that as a separate PR if anyone feels so inclined.

Reply:
@npauzenga How do you feel if I made changes to `create` to not return an array in a separate PR? Updating the function return type has major implications for lines of code outside of what my PR handles and I would rather keep those changes in a separate PR.

Reply:
To be clear, I'm on board with @npauzenga's suggestion to update all of our models to return `*` fields from the `create` method, but we all agree that should be done in a separate PR

I'm less convinced that we can/should refactor `create` to return an object vs an array. I can see the minor convenience of not needing to destructure the return value, but is there any value beyond that? It would also require refactoring a lot of affected code. 

The other thing about both of these suggestions is that neither one is really required for us to move forward and at some point we need to start limiting the scope of our work in order to complete RDS phase 2 work in a timely fashion.

Reply:
Totally on board with not doing any of it in this PR to limit scope and get it out faster. It also doesn't have to be something you handle @botanical. It's really easy for me to pop into someone else's PR and request something that's tangential but not totally relevant to the work 😄